docs/getting-started.md

```diff
@@ -265,38 +265,46 @@ on a bigger project you will see that `ScanCode` often finds more licenses than
 ## 6. Running the evaluator
 
 The evaluator can apply a set of rules against the scan result created above. ORT provides a
-[sample rules file](../evaluator/src/main/resources/rules/no_gpl_declared.kts) on the classpath that can be used for
-testing the evaluator. The sample rule creates an error if any package contained in the result declares a GPL license.
-To run the sample rules use:
+[example policy rules file](examples/rules.kts) that can be used for testing the `evaluator`. 
+
+To run the example rules use:
 
 ```bash
-cli/build/install/ort/bin/ort evaluate --rules-resource rules/no_gpl_declared.kts \
-    -i [scanner-output-path]/scan-result.yml -o [evaluator-output-path]/mime-types
+cli/build/install/ort/bin/ort evaluate
+  --rules-file rules.kts
+  -i [scanner-output-path]/scan-result.yml
+  -o [evaluator-output-path]/mime-types
 ```
 
-Because neither mime-types nor any of its dependencies declares a GPL license this finishes without an error.
-
-It is possible to write your own evaluator rules as Kotlin script and pass it to the evaluator using `--rules-file`.
+It is possible to write your own evaluator rules as Kotlin script and pass it to the `evaluator` using `--rules-file`.
 Note that detailed documentation for writing custom rules is not yet available.
 
 ## 7. Generate a report
 
-The `evaluation-result.yml` file can now be used as input for the reporter to generate human-readable reports. For
-example, to generate both a static HTML report and an Excel report use:
+The `evaluation-result.yml` file can now be used as input for the reporter to generate human-readable reports
+and open source notices. For
+example, to generate both a static HTML report, WebApp report and open source notice by package use:
```

Comment:
example, to generate both a static HTML report, WebApp report and open source notice by package use: --> example, to generate a static HTML report, WebApp report and an open source notice by package, use:

Reply:
done